com.unity.render-pipelines.high-definition/Runtime/Lighting/GlobalIllumination.cs

```diff
@@ -50,24 +50,6 @@ public int raySteps
         [Tooltip("Controls the number of steps used for ray marching.")]
         private ClampedIntParameter m_RaySteps = new ClampedIntParameter(24, 16, 128);
 
-        /// <summary>
```

Comment:
This will be considered a breaking change. Not sure if we can use the [Obsolete] attribute on volume parameters. Would need to check that. @sebastienlagarde can we consider doing it anyway if it's for version 10.x.x anyway?

Reply:
GlobalIllumination is officially  in preview, so i think we can break it

Reply:
yes, SSGI was in preview up to now (and have been added in 10.x anyway), btw once we merge this PR we should remove preview from SSGI UI.